.editorconfig

```diff
@@ -0,0 +1,30 @@
+# http://editorconfig.org
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+
+[*.{html,json}]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = false
+
+[*.{scss,js}]
```

Comment:
No particular reason for tab versus space other than likely convenience for code to conform to existing tab settings. If there are already style guidelines for the JS and CSS, then I would say it would be best to adhere to those. indent_style = space, indent_size = 2 Would that be better?

Since SCSS is not used, I agree it is best to remove.